workflow/scripts/osemosys_global/summary/headline.py

```diff
@@ -0,0 +1,121 @@
+"""Calculates Headline Metrics"""
+
+import pandas as pd
+from constants import RENEWABLES, FOSSIL, CLEAN
+
+
+def get_emissions(annual_emissions: pd.DataFrame) -> pd.DataFrame:
+
+    df = annual_emissions.copy()
+
+    total_emissions = df.VALUE.sum().round(0)
+    data = ["Emissions", "Million tonnes of CO2-eq.", total_emissions]
+    return pd.DataFrame([data], columns=["Metric", "Unit", "Value"])
+
+
+def get_system_cost(total_discounted_cost: pd.DataFrame) -> pd.DataFrame:
+
+    df = total_discounted_cost.copy()
+
+    total_cost = round((df.VALUE.sum() / 1000), 0)
+
+    data = ["Total system cost", "Billion $", total_cost]
+    return pd.DataFrame([data], columns=["Metric", "Unit", "Value"])
+
+
+def get_gen_cost(
+    total_discounted_cost: pd.DataFrame, demand: pd.DataFrame
+) -> pd.DataFrame:
+
+    costs = total_discounted_cost.copy()
+    dem = demand.copy()
+
+    total_cost = costs.VALUE.sum()  # $M
+    total_demand = dem.VALUE.sum()  # PJ
+
+    # ($M / PJ) ($1000000 / $M) (1 PJ / 1000 TJ) (1 TJ / 1000 GJ) (1 GJ / 1000 MJ) (3600sec / 1000 hr) = 3.6
+    gen_cost = ((total_cost / total_demand) * 3.6).round(0)
+
+    data = ["Cost of electricity", "$/MWh", gen_cost]
+    return pd.DataFrame([data], columns=["Metric", "Unit", "Value"])
+
+
+def _filter_pwr_techs(production_by_technology: pd.DataFrame) -> pd.DataFrame:
+
+    df = production_by_technology.copy()
+
+    return df[
+        (df.index.get_level_values("TECHNOLOGY").str.startswith("PWR"))
+        & ~(df.index.get_level_values("TECHNOLOGY").str.startswith("PWRLDS"))
```

Comment:
This should be read dynamically from the storage_parameters entry in the config file since the storage technologies can change.